contracts/release1/build1/build-metadata-R1B1.json

```diff
@@ -0,0 +1,21 @@
+{
+  "ui": "",
```

Comment:
Same as on the other PR, I recommend leaving this field as an object for future proofness. I can see many keys being part of the UI section